scripts/populateDrupalCache.py

```diff
@@ -2,16 +2,33 @@
 
 import requests
 from xml.dom import minidom
+import sys
 
-sitemaps = [
-    'https://access.redhat.com/sitemap/assembly.sitemap.xml',
-    'https://access.redhat.com/sitemap/module.sitemap.xml'
-]
+env = dict({
+        "prod" : [
+            'https://access.redhat.com/sitemap/assembly.sitemap.xml',
+            'https://access.redhat.com/sitemap/module.sitemap.xml'
+        ],
+        "qa" : [
+            'https://pantheon.corp.qa.redhat.com/api/sitemap/assembly.sitemap.xml',
+            'https://pantheon.corp.qa.redhat.com/api/sitemap/module.sitemap.xml'
+        ],
+        "stage" : [
+            'https://pantheon.corp.stage.redhat.com/api/sitemap/assembly.sitemap.xml',
+            'https://pantheon.corp.stage.redhat.com/api/sitemap/module.sitemap.xml'
+        ],
+        "dev" : [
+            'https://pantheon.corp.dev.redhat.com/api/sitemap/assembly.sitemap.xml',
+            'https://pantheon.corp.dev.redhat.com/api/sitemap/module.sitemap.xml'
+        ]})
 
-def gather_urls():
+headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36'}
```

Comment:
Query: Do we need headers?

Reply:
@aprajshekhar It is not helping us, i kept to to get rid off forbidden 403. But on preprod, we are still getting it. So we can remove it.